### PR TITLE
Don't use Zstandard bundles on Windows

### PR DESCRIPTION
**Pull request by henrymercer**
In testing, gzip performs better than Zstandard on Windows.

### Merge / deployment checklist

- [ ] Confirm this change is backwards compatible with existing workflows.
- [ ] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [ ] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

File: pr-checks/checks/zstd-bundle.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ versions:
   - linked
 operatingSystems:
   - macos
-  - windows
```
Comment (aeisenberg) [inline pr-checks/checks/zstd-bundle.yml:7]: Do we have a test that ensures zstd is not used on windows?

**Review by aeisenberg — COMMENTED** [review]
Wild...I didn't expect that.

**Review by aeisenberg — APPROVED** [review]
Nice.

File: pr-checks/checks/zstd-bundle.yml (hunk 1 above)
Comment (henrymercer) [reply pr-checks/checks/zstd-bundle.yml:7]: Let's add one :)

**Pull request merged**